rules/falco_rules.yaml

```diff
@@ -1910,14 +1910,19 @@
 - macro: grep_commands
   condition: (proc.name in (grep_binaries))
 
+- macro: grep_more
```

Comment:
Can you add a comment indicating what this does (namely a less restrictive search for things that might be passwords/ssh/etc) ?